package.json

```diff
@@ -98,6 +98,7 @@
     "terra-enzyme-intl": "^3.0.0",
     "terra-toolkit": "^6.2.0",
     "url-loader": "^4.1.1",
+    "uuid": "^8.3.2",
```

Comment:
Unfortunately, I think we need to use v3 like we do in terra-application: https://github.com/cerner/terra-application/blob/main/package.json#L127

v8 isn't compatible with IE10 which is needed for MPages.

Reply:
Removed the commit.